### ISSUE TITLE
"No Information Received" into AND column?

### ISSUE THREAD

**Issue by vaderag** [daneedk/com.uc.heimdall#28]
Would there be a possibility of sticking the “No Information Received” card in the And column?

Reason being I’d like to do something like:
When the time is 7.30pm, and No Information Received and Last Update 24hours and device is not xyz, then send an alert

Basically, so I can get the alerts at the same time every day, when I generally have some way of actioning it?

**Comment by daneedk** [comment]
I've looked into this and brainstormed on how to best implement this. Adding a condition card is not the way I want to go, that would require quite a large change in Heimdall and would also not scale well when there are a lot of devices connected to Homey. 

So I decided to go for an action card that would enable you the run the **Check Last Communication** function at your desired condition in the same way it is now run at setting a **Surveillance Mode**.
That will result in the same action that's already in place in **Heimdall** so writing a line to the history and triggering a flowcard.

So basically if you have this flow now, that gets triggered at setting the **Surveillance Mode**: 
![image](https://user-images.githubusercontent.com/16610696/47604013-4e144900-d9f4-11e8-8934-75c5baca9c36.png)

You will be able to trigger it with this new flow using the new **Check Last Communication** card:
![image](https://user-images.githubusercontent.com/16610696/47604049-a3505a80-d9f4-11e8-82e7-3450c46c590d.png)

This new flow card will be available in version 1.0.26.
(Current version is 1.0.24, version 1.0.25 is pending approval by Athom)

**Comment by daneedk** [comment]
While testing over a longer period I discovered a nasty bug in the communication check which I want to fix before releasing this flowcard.

**Comment by daneedk** [comment]
I've created a bug report with Athom but no response yet...
https://github.com/athombv/homey/issues/2917

**Comment by daneedk** [comment]
During some testing this function seems to work good on V2 and the bug that seems to be in 1.5.13 seems to be absent in 2.0.

This function can be started in 2 ways: By setting the **Surveillance Mode** or by using the action card **Check Last Communication**.
When the function is executed the result will be written to the **History** and to the **Timeline** (optional). 
Also the trigger card **No information received** card is triggered with the tags **Hours no information**, **Last Update**, **Device** and **Zone** so you can use a method of your choice to inform you.
